.travis.yml

```diff
@@ -5,4 +5,5 @@ script:
   - cd website; yarn install; yarn build
   - npx serve build/near-docs &
   - npx wait-on http://localhost:5000/
-  - npx broken-link-checker -gro --exclude "localhost:3030" --exclude "https://github.com/nearprotocol/docs/tree/master/docs" --exclude "https://youtu.be" --exclude "http://near.ai/wbs" --exclude "https://www.youtube.com" --host-requests 2 http://localhost:5000 --exclude "https://rpc.testnet.near.org" --exclude "https://rpc.betanet.near.org" --exclude "https://rpc.near.org"  --exclude "https://crates.io/crates/near-sdk"  --exclude "https://github.com/nearprotocol/workshop" --exclude "https://wasmbyexample.dev"
\ No newline at end of file
+  - npx broken-link-checker -gro --exclude "localhost:3030" --exclude "https://github.com/nearprotocol/docs/tree/master/docs" --exclude "https://youtu.be" --exclude "http://near.ai/wbs" --exclude "https://www.youtube.com" --host-requests 2 http://localhost:5000 --exclude "https://rpc.testnet.near.org" --exclude "https://rpc.betanet.near.org" --exclude "https://rpc.near.org"  --exclude "https://crates.io/crates/near-sdk"  --exclude "https://github.com/nearprotocol/workshop" --exclude "https://wasmbyexample.dev" --exclude "https://near.zavodil.ru/?pools=" --exclude "https://staking.dokia.cloud/staking/near/validators" --exclude "https://www.timeanddate.com/moon/phases/"
```

Comment:
wait, why doesn't this run `yarn test`??

Reply:
we should also add `broken-link-checker` as a dependency, instead of pulling the latest version. Pulling the latest version on every push will break our CI if `broken-link-checker` gets updated in a way that isn't backwards compatible

Reply:
> wait, why doesn't this run `yarn test`??

I'm not sure. I did not originally set this up.